segmentstore/storage/impl/src/main/java/io/pravega/segmentstore/storage/impl/bookkeeperstorage/BookKeeperStorageConfig.java

```diff
@@ -0,0 +1,142 @@
+/**
+ * Copyright (c) 2017 Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.pravega.segmentstore.storage.impl.bookkeeperstorage;
+
+import io.pravega.common.util.ConfigBuilder;
+import io.pravega.common.util.ConfigurationException;
+import io.pravega.common.util.InvalidPropertyValueException;
+import io.pravega.common.util.Property;
+import io.pravega.common.util.TypedProperties;
+import java.nio.charset.Charset;
+import java.time.Duration;
+import java.util.Arrays;
+import lombok.Getter;
+
+public class BookKeeperStorageConfig {
+    public static final Property<String> ZK_ADDRESS = Property.named("zkAddress", "localhost:2181");
+    public static final Property<Integer> ZK_SESSION_TIMEOUT = Property.named("zkSessionTimeoutMillis", 10000);
+    public static final Property<Integer> ZK_CONNECTION_TIMEOUT = Property.named("zkConnectionTimeoutMillis", 10000);
+    public static final Property<String> ZK_METADATA_PATH = Property.named("zkMetadataPath", "/segmentstore/bookkeeperstorage/");
+
+    public static final Property<Integer> BK_ENSEMBLE_SIZE = Property.named("bkEnsembleSize", 3);
+    public static final Property<Integer> BK_ACK_QUORUM_SIZE = Property.named("bkAckQuorumSize", 3);
+    public static final Property<Integer> BK_WRITE_QUORUM_SIZE = Property.named("bkWriteQuorumSize", 3);
+    public static final Property<Integer> BK_WRITE_TIMEOUT = Property.named("bkWriteTimeoutMillis", 5000);
```

Comment:
I recently added a Read Timeout (check out (an updated) `BookKeeperConfig` for inspiration). I would like this here too because the default timeout is 5s, which may be inappropriate in some cases.

Reply:
Fixed.